### PR TITLE
Comment requirements on concrete types

### PR DESCRIPTION
**Pull request by alasdairlaw**
* Clarify that implementations of interfaces do not require comments if the interface already has it written

### CODE REVIEW

**Comment by alasdairlaw** [comment]
Asking for another review of this for people who already approved it, since it has changed.

File: Readme.md [hunk 1]
```diff
@@ -102,6 +102,7 @@ Properties ordered and separated by mutability and type of reference (constants,
 * All public APIs for frameworks should be documented, and all internal APIs for apps should be documented.
 * All types should at least have a comment above the type definition to explain what it's used for.
 * Internal / private APIs should be commented where there's business logic that isn't already sufficiently explained by method and variable naming.
+* If a property or function is defined by an interface and implemented by a concrete type, the concrete types must not rewrite the comment. If changes need to be made to the documentation for that concrete type, the concrete type by definition does not follow that interface any more. If comments need to be made about the implementation detail for that concrete type they can be done as inline comments that are only available internally.
```
[the remark this answers is not in the document]
Comment (alasdairlaw) [reply Readme.md]: Sure, I think it says pretty much the same thing, but I'll use that

**Comment by alasdairlaw** [comment]
@bartszczepaniak ?

**Comment by RobbieDC** [comment]
Agreed, if the code doesn't explain itself it should be rewritten in a way that it does.

**Comment by Flinted** [comment]
Personally I would agree.  

The main app is not going to be imported elsewhere whereas Frameworks likely are.

**Comment by Andrew-Walker** [comment]
I literally couldn't agree more. Excellent points made by all!

**Comment by alasdairlaw** [comment]
Closing to amend this for public comments in apps - will reopen so everyone can review again.

**Pull request closed without merging**